### PR TITLE
Moscow Domodedovo fixes and traffic

### PR DESCRIPTION
**Pull request by culebron**
Improved Moscow Domodedovo airport data, added all possible fixes and navaids.

### CODE REVIEW

File: assets/airlines/tso.json [hunk 1]
```diff
@@ -1,18 +0,0 @@
-{
```
Comment (culebron) [inline assets/airlines/tso.json:1]: This company went bankrut last month.

**Comment by culebron** [comment]
@glangford  @nielsvdweide @harp71 please test this when you have time.

**Comment by glangford** [comment]
> @glangford @nielsvdweide @harp71 please test this when you have time.

:+1: It's great to have a comprehensive set of fixes with lat/long (and updating the set of airlines). For the airports I have worked on I haven't focused on getting the airlines right for sure.

One comment - the altitudes of arrivals start quite low?

**Comment by culebron** [comment]
@glangford  in the last update with restricted areas branch I raised them to 8K..10K. Don't remember why I did not push that here.

**Pull request merged**